src/shared/utils/logger.go

```diff
@@ -0,0 +1,41 @@
+package utils
+
+import (
+	"log"
+	"os"
+)
+
+// file path of our logs
+var path = "logs/logs.txt"
```

Comment:
I suggest that we name out log file “anusic-api.log”

Reply:
hmmm ok